src/utils/config.ts

```diff
@@ -70,7 +70,19 @@ export const MAXIMUM_DRAIN_DELAY = Math.round(
 	(1 * MINUTES) / intVar('MINIMUM_DRAIN_RATE_PER_MINUTE', 500),
 );
 
-export const VPN_INSTANCE_COUNT = getInstanceCount('VPN_INSTANCE_COUNT');
+export let VPN_INSTANCE_COUNT = getInstanceCount('VPN_INSTANCE_COUNT');
+const VPN_OPENVPN_PROTO = optionalVar('VPN_OPENVPN_PROTO');
+if (VPN_OPENVPN_PROTO) {
+	if (/^udp[4-6]?$/.test(VPN_OPENVPN_PROTO)) {
+		if (VPN_INSTANCE_COUNT > 1) {
+			console.error(
```

Comment:
I wonder if it would be better for this to just straight up error rather than quietly falling back to single process, because if you're expecting eg 4 processes per instance and you only get 1 process per instance you might hit scaling issues and might not immediately realize if you look at cpu stats and see eg 25% cpu usage that it's because there's only 1 process and not the 4 you expected.